pkg/torcx/paths.go

```diff
@@ -26,11 +26,14 @@ const (
 	DefaultBaseDir = "/var/lib/torcx/"
 	// DefaultConfDir is the default torcx config directory
 	DefaultConfDir = "/etc/torcx/"
+
 	// VendorStorePath is the vendor store path
-	VendorStorePath = VENDOR_DIR + "/store/"
+	VendorStorePath = VendorDir + "store/"
+	// VendorProfile is the vendor (lower) profile
+	VendorProfile = VendorDir + VendorProfileName + ".json"
```

Comment:
Good question. I think we may want to keep "profiles/" as a place where we store all profiles (in case we decide to ship several pre-built, eg. for tectonic), and then symlink `vendor.json` to it. Makes sense?